lib/WP/Functions.php

```diff
@@ -534,7 +534,23 @@ function wpautop($pee, $br = true) {
    */
   function parseDbHost($host) {
     global $wpdb;
-    return $wpdb->parse_db_host($host);
+    if (method_exists($wpdb, 'parse_db_host')) {
+      return $wpdb->parse_db_host($host);
+    } else {
+      // Backward compatibility for WP 4.7 and 4.8
+      $db_host = DB_HOST;
```

Comment:
Why do you use the `DB_HOST` constant and not the `$host` function argument?

Reply:
Ohh. Good catch! It was a copy paste from old version. I did not realized that. Will fix it.